src/api/cvc4cpp.cpp

```diff
@@ -3048,6 +3048,13 @@ Term Solver::mkTuple(const std::vector<Sort>& sorts,
 /* Create operators                                                           */
 /* -------------------------------------------------------------------------- */
 
+Op Solver::mkOp(Kind kind) const
```

Comment:
Can you add a unit test?

Reply:
Definitely! I'll add that now.

I actually just realized there's an issue here. Adding this makes it kinda easy to do stupid things, such as: `mkOp(BITVECTOR_EXTRACT)`. How would you recommend guarding against that? Should we just list all indexed operators and check that it's not one of those? Or would it be better to just not have this `mkOp` with a single `Kind` functionality?

Reply:
Good question. If we keep it, we should guard against using it with indexed operators. We don't add new operators frequently, so I think that should be fine.

Reply:
Sounds good. Made that change with 9c8f7e7d186c08db8f190556cf5797d1a677f081 (+ a small fix in the subsequent commit)